### PR TITLE
Expands ORM accepted accesses

### PR DESCRIPTION
**Pull request by Anonmare**
[Changelogs]: The ORM machine will now accept accesses from other departments, meaning you don't have to pester cargo or break the machine to fill your department lathes any more. Most access requirements are generic - bar security's, considering their lathe's the one with the weaponry.

:cl:
balance: The ORM is now accessible to a wider range of people
/:cl:

[why]: Honestly I was just going to remove the access altogether since the machine only needs a screwdriver and a crowbar to deconstruct and get at the materials but this is a comprimise.

### CODE REVIEW

File: code/modules/mining/machine_redemption.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	anchored = TRUE
 	input_dir = NORTH
 	output_dir = SOUTH
-	req_access = list(ACCESS_MINERAL_STOREROOM)
+	req_access = list(ACCESS_MINERAL_STOREROOM, ACCESS_MEDICAL, ACCESS_ARMORY, ACCESS_CONSTRUCTION)
```
Comment (ACCount12) [inline code/modules/mining/machine_redemption.dm]: Wouldn't that make it require ALL listed access types?

Comment (Anonmare) [reply code/modules/mining/machine_redemption.dm]: I've been asking in Discord and the one person who's answered didn't know. The Release option's code is as follows:

>if(check_access(inserted_id) || allowed(usr)) //Check the ID inside, otherwise check the user
                if(params["id"] == "all")
                    materials.retrieve_all(get_step(src, output_dir))
                else

Not sure if this means it's looking for every access or only a singular one

Comment (ACCount12) [reply code/modules/mining/machine_redemption.dm]: Just test it ingame. If it wouldn't work as you intend it to, there's `req_one_access` var.

File: code/modules/mining/machine_redemption.dm [hunk 2]
```diff
@@ -257,7 +257,7 @@
 		if("Release")
 
 			if(check_access(inserted_id) || allowed(usr)) //Check the ID inside, otherwise check the user
-				if(params["id"] == "all")
+				if(params["id"] == "any")
```
Comment (ACCount12) [inline code/modules/mining/machine_redemption.dm]: Why this change?

**Comment by Anonmare** [comment]
I'll come back when I figure this old code out
